source/Core/Inc/Settings.h

```diff
@@ -47,8 +47,8 @@ enum SettingsOptions {
   PDMissingWarningCounter        = 30, // Counter of how many times we have warned we cannot detect the pd interface
   UILanguage                     = 31, // Selected UI Language code, null-terminated *only if* the length is less than 8 chars
   PDNegTimeout                   = 32, // PD timeout in 100ms steps
-  OLEDInversion                  = 33, // Invert the colours on the display
```

Comment:
Can you undo this swap please, need to keep setting <-> number the same to support settings migration on updates :)

Reply:
Done. 😊